src/SectorView.cpp

```diff
@@ -764,29 +764,38 @@ void SectorView::PutFactionLabels(const vector3f &origin)
 
 				Gui::Screen::MeasureString(labelText, labelWidth, labelHeight);
 
-
-				{
-					Graphics::VertexArray va(Graphics::ATTRIB_POSITION);
-					va.Add(vector3f(pos.x - 5.f,              pos.y - 5.f,               0));
-					va.Add(vector3f(pos.x - 5.f,              pos.y - 5.f + labelHeight, 0));
-					va.Add(vector3f(pos.x + labelWidth + 5.f, pos.y - 5.f,               0));
-					va.Add(vector3f(pos.x + labelWidth + 5.f, pos.y - 5.f + labelHeight, 0));
-					m_material->diffuse = labelBorder;
-					m_renderer->DrawTriangles(&va, renderState, m_material.Get(), Graphics::TRIANGLE_STRIP);
-				}
```

Comment:
You removed the explicit code blocks, any reason?

Reply:
any reason for the codeblocks being there in the first place? other than not having to rename the variable?

I dont know if the compiler optimizes away locally created stack frames, but if it doesnt, then thats overhead just for being too lazy to rename a variable.

Reply:
1. If the code-snippet is repeated, with just variable name change, then why not a nice lambda-function?

2. putting actual feature change/addiont in seperate commit, and then all refactoring changes in a single commit e.g. names "refactoring changes" or "indentation fix" or something, makes it easier to review.

Reply:
Dude... I dont know C++.. but I do know that the new code is in 3 blocks that replace the previous two blocks. none of the three new blocks is identical, they all have some differences. none of the change is "indentation fix".